partners.html

```diff
@@ -0,0 +1,100 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="/style.css">
+  <link rel="stylesheet" href="/index.css">
+  <title>About-page</title>
+</head>
+<body>
+  <header class="header-container" >
+    <div class="first-section-nav">
```

Comment:
- [ ] It would be nice to use the `<nav>` tag instead of the `<div>` tag used here for semantic purposes.